tests/integ/scala/test_async_job_suite.py

```diff
@@ -349,13 +351,21 @@ def test_async_batch_insert(session):
     reason="TODO(SNOW-932722): Cancel query is not allowed in stored proc",
 )
 def test_async_is_running_and_cancel(session):
-    async_job = session.sql("select SYSTEM$WAIT(3)").collect_nowait()
+    # creating a sproc here because describe query on SYSTEM$WAIT()
```

Comment:
the commit is checked in accidentally?

Reply:
no. this is another one of the case where describe query is launched. in this case, internal describe query on `SELECT$WAIT` actually blocks the query

Reply:
I see, but I mean should it be in another PR?

Reply:
+ 1 this change seems irrelevant to this pr

Reply:
I will revert this change and fix SelectSQL implementation so we do not trigger analyze attributes for it.